### PR TITLE
Fix some errors

### PR DESCRIPTION
**Pull request by PaulMansat**
- fix error in training alone (wasn't using the proper compile data)
- fix the asynchronous behavior of the  creation of models

### CODE REVIEW

**Comment by PaulMansat** [comment]
As soon as these changes are validated, I will merge them. 
This will fix the error we had with models with static performance after the 1st epoch. 
@MarcelTorne these enhancements might require you to change the optimizer defined in the LUS task (I saw that in the alone training loop the Adam optimizer was hardcoded, which is not the case anymore).

**Comment by PaulMansat** [comment]
This will also fix the error we had when: 
(1) User fist log into the app 
(2) Joins the training of task 
(3) Error "Could not find the model working_[taskName]" was returned 

This is now fixed

File: mobile-browser-based-version/src/task_definition/mnist.js [hunk 1]
```diff
@@ -240,7 +240,7 @@ export const trainingInformation = {
     // {String} model's identification name
     modelId: "mnist-model",
     // {String} indexedDB path where the model is stored
-    savePathDb: "indexeddb://working_lus_covid_model",
+    savePathDb: "indexeddb://working_mnist-model",
```
Comment (martinjaggi) [inline mobile-browser-based-version/src/task_definition/mnist.js:243]: underscore?

File: mobile-browser-based-version/src/task_definition/lus_covid.js [hunk 2]
```diff
@@ -253,7 +253,7 @@ export const trainingInformation = {
     // {String} model's identification name
     modelId: "lus-covid-model",
     // {String} indexedDB path where the model is stored
-    savePathDb: "indexeddb://working_lus_covid_model",
+    savePathDb: "indexeddb://working_lus-covid-model",
```
Comment (martinjaggi) [inline mobile-browser-based-version/src/task_definition/lus_covid.js:256]: ok can say it's working_+id

Comment (PaulMansat) [reply mobile-browser-based-version/src/task_definition/lus_covid.js:256]: Right 👍

**Pull request merged**